addons/account/views/report_invoice.xml

```diff
@@ -19,30 +19,30 @@
                         <span t-field="o.number"/>
                     </h2>
 
-                    <div class="row mt32 mb32">
-                        <div class="col-auto" t-if="o.name" name="description">
+                    <div id="informations" class="row mt32 mb32">
+                        <div class="col-auto mt-2 mb-2" t-if="o.name" name="description">
```

Comment:
`my-2` would be better but ok

Reply:
I didn't know it exists, thanks for the tip.